hal/src/electron/ota_flash_hal.cpp

```diff
@@ -43,5 +43,6 @@ void HAL_OTA_Add_System_Info(hal_system_info_t* info, bool create, void* reserve
 
         set_key_value(info->key_values + count, "imei", device.imei);
         set_key_value(info->key_values + count + 1, "iccid", device.iccid);
+        set_key_value(info->key_values + count + 2, "radiofw", device.radiofw);
```

Comment:
Let's be more specific in what this means, perhaps? `radiofw` sounds way too generic. Which radio? WiFi? BLE? Cellular? (Tracker has all of them and all of them have a version). Same in other places and where this info is presented through CLI (system_setup.cpp).

Reply:
Since it belongs to 'Cellular'Device already, I left it radiofw. For example, Wifi devices might also have an imei? No strong preference, but just want it not be redundant with `cellradiofw` field with in `CellularDevice`? 🤔  This string can change though - https://github.com/particle-iot/device-os/pull/2265/files#diff-b43738c13eafa76322b3d732450e7d9838fac9ffadda001a79653beafb5b93bfR282

Reply:
The field in `CellularDevice` is fine, but as you can see below in your describe messages that you've posted, ICCID, IMEI and now `radiofw` are top level citizens in that json object.

Reply:
I changed it to `cellradiofw` but honestly, better name ideas welcome... (I will post results on develop to make sure size looks correct)